pkg/driver/state.go

```diff
@@ -16,24 +16,34 @@ const (
 	// StateRunning means that the driver has been sending data. The caller
 	// who started the driver may start reading data from the hardware.
 	StateRunning = "running"
+	// StateGood means the driver is available for use by applications.
+	StateGood = "good"
```

Comment:
Because this is now dependent on a specific driver it's a lot more difficult to test. For Linux devices this requires an ioctls call. Short of mocking the return values for ioctls or setting up a virtual v4l2 device I don't think there's a good way to test it. This is also a public function that isn't needed much internally so you wouldn't be able to mock out the `Status` function and see the behavior of internal functions change either. It's tricky.